src/sc_crawler/inspector.py

```
@@ -101,10 +103,22 @@ def _server_path(server: "Server") -> str | PathLike:
 
+def _get_server_framework_run_ids(server: "Server", framework: str) -> List[str]:
+    return [
+        p.name
+        for p in Path(_server_framework_path(server, framework)).iterdir()
+        if p.is_dir()
+    ]
+
+
 def _server_framework_path(
-    server: "Server", framework: str, relpath: str = None
+    server: "Server", framework: str, relpath: str | list[str] = None
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)